sources/packages/backend/apps/db-migrations/src/migrations/1706733349802-RemoveConstraintStudentAssessmentsOnlyOneAssessmentDataNullIndex.ts

```diff
@@ -0,0 +1,24 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { getSQLFileData } from "../utilities/sqlLoader";
+
+export class RemoveConstraintStudentAssessmentsOnlyOneAssessmentDataNullIndex1706733349802
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      getSQLFileData(
+        "Drop-index-only-one-assessment-data-null.sql",
+        "StudentAssessments",
+      ),
+    );
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      getSQLFileData(
+        "Create-index-only-one-assessment-data-null.sql",
```

Comment:
The changes look good. In a previous PR there was a conversation about creating the `down` script adding a `Rollbak` prefix to it, as below. Can we follow it, please?
- Drop-index-only-one-assessment-data-null.sql
- Rollback-drop-index-only-one-assessment-data-null.sql

Reply:
Sure. I'll follow the prefix.